app/jobs/html_variant_success_create_job.rb

```diff
@@ -0,0 +1,7 @@
+class HtmlVariantSuccessCreateJob < ApplicationJob
+  queue_as :html_variant_success_create
+
+  def perform(hash)
```

Comment:
Why not use explicit keyword arguments? 

Something like:

```ruby
def perform(html_variant_id:, article_id:)
```

Reply:
It seemed to me it was shorter and it won't be necessary to change the signature of perform as well as the call under in case of an update of the HtmlVariantSuccess model.

Though, I should have used "attributes" instead of "hash", since from the Ruby api doc, one can pass either a Hash or an Array.

So I used explicit arguments instead.